doc/content/integrations/storage/configuration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configuration
+description: ""
+weight: 10
+---
+
+The Storage Integration requires configuration for the underlying storage provider used, along with a few options for tuning performance and memory usage.
+
+You can configure PostgreSQL, or any other PostgreSQL-compatible database (e.g. [TimescaleDB](https://www.timescale.com/)). A base-line configuration can be found below. Add the following section into your existing `ttn-lw-stack.yml` configuration file. See [Storage Integration Options]({{< ref "/reference/configuration/application-server#storage-integration-options" >}}) for more details.
+
+```yaml
+# ttn-lw-stack.yml
+
+as:
```

Comment:
Add this to `getting-started/installation/configuration/docker-compose-enterprise.yml`
and you should be able to do `{{< highlight yaml "linenos=table,linenostart=5" >}} {{< readfile path="/content/getting-started/installation/configuration/docker-compose-enterprise.yml" from=5 to=14 >}} {{< /highlight >}}`

see https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/getting-started/installation/configuration/_index.md#sql-database